stress/src/etw_logs.rs

```diff
@@ -0,0 +1,66 @@
+//! Run this stress test using `cargo run --bin etw --release -- <num-of-threads>`.
+//!
+// Conf - AMD EPYC 7763 64-Core Processor 2.44 GHz, 64GB RAM, Cores:8 , Logical processors: 16
+// Stress Test Results (no listener)
+// Threads: 1 - Average Throughput: 52M iterations/sec
+// Threads: 5 - Average Throughput: 250M iterations/sec
+// Threads: 10 - Average Throughput: 320M iterations/sec
+// Threads: 16 - Average Throughput: 400M iterations/sec
+
+// Stress Test Results (logman is listening)
+// Threads: 1 - Average Throughput: 600K iterations/sec
+// Threads: 5 - Average Throughput: 2.7M iterations/sec
+// Threads: 10 - Average Throughput: 4.1M iterations/sec
+// Threads: 16 - Average Throughput: 5M iterations/sec
```

Comment:
These seems to be much better than user_events, which are below:
```
// Stress Test Results (user_events enabled)
// Threads: 1 - Average Throughput: 285,692 iterations/sec
// Threads: 5 - Average Throughput: 392,906 iterations/sec
// Threads: 10 - Average Throughput: 349,334 iterations/sec
// Threads: 16 - Average Throughput: 297,232 iterations/sec
```

Reply:
Yes! Hopefully, https://github.com/open-telemetry/opentelemetry-rust-contrib/issues/159 should make it comparable to ETW!